pkg/vgmanager/vgmanager_controller.go

```diff
@@ -46,7 +46,7 @@ const (
 func (r *VGReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	r.deviceAgeMap = newAgeMap(&wallTime{})
 	return ctrl.NewControllerManagedBy(mgr).
-		For(&lvmv1alpha1.LVMCluster{}).
+		For(&lvmv1alpha1.LVMVolumeGroup{}).
```

Comment:
Will creation of a new `LVMVolumeGroup` trigger controller reconcile for all the other existing LVMVolumeGroups?
If yes, then I think we should avoid that.